src/smartthings/smartapps/smartthings/amazon-echo-custom.src/amazon-echo-custom.groovy

```diff
@@ -131,38 +151,38 @@ preferences(oauthPage: "oauthPage") {
         }
     }
 
-    // Instructions for the user on how to run appliance discovery on Echo to update its device list
+    // Instructions for the user on how to run appliance discovery from Alexa to update its device list
     // install: true is needed here to have a Done button which will return user to SmartApp list.
-	page(name: "instructionPage", title: "Device Discovery", install: true) {
-		log.trace "instructionPage pref page"
-		section("") {
-			paragraph "You have made a change to your device list.\n\n" +
-					"Now complete device discovery by saying the following to your Echo:"
-			paragraph title: "\"Alexa, discover new devices\"", ""
-		}
-	}
-
-    page(name: "installPage", title: "Install", install: true) {
-        section("") {
-			paragraph '''\
-This is the install page:
-
-We would have actually redirected to the provisioning URL here.
-But just tap Done in the upper right to install the app and pretend.'''
-		}
+    page(name: "instructionPage", content: "buildInstructionPage", install: true)
+
+    page(name: "helpPage", title: "Help") {
+        section("Knowledgebase Articles") {
+            href(
+                name: "href",
+                title: "Amazon Alexa",
+                url: "https://support.smartthings.com/hc/en-us/articles/205275404",
+                style: "embedded"
+            )
+            href(
+                name: "href",
+                title: "SmartThings + Amazon Alexa FAQ",
+                url: "https://support.smartthings.com/hc/en-us/articles/207808076",
+                style: "embedded"
+            )
+        }
     }
 
-	// Separate page for uninstalling, we dont want the user to accidentaly uninstall since the app can only be automatically reinstalled
-	page(name: "uninstallPage", title: "Uninstall", uninstall: true, nextPage: "deviceAuthorization") {
-		// log.trace "uninstallPage pref page"
-		section("") {
-			paragraph '''\
-If you uninstall this SmartApp, remember to unlink your SmartThings account from Echo:
-  1. Open the Amazon Echo application
+    // Separate page for uninstalling, we dont want the user to accidentaly uninstall since the app can only be automatically reinstalled
+    page(name: "uninstallPage", title: "Uninstall", uninstall: true, nextPage: "deviceAuthorization") {
+        // log.trace "uninstallPage pref page"
+        section("") {
+            paragraph '''\
+If you uninstall this SmartApp, remember to unlink your SmartThings account from Alexa:
+  1. Open the Amazon Alexa application
   2. Goto Settings > Connected Home > Device Links
```

Comment:
Updated, but we will have to have this copy reviewed as a whole prior to release